ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/BySeveritySummaryCard.tsx

```diff
@@ -5,19 +5,26 @@ import SeverityIcons from 'Components/PatternFly/SeverityIcons';
 
 import { VulnerabilitySeverity } from 'types/cve.proto';
 import { vulnerabilitySeverityLabels } from 'messages/common';
+import { SVGIconProps } from '@patternfly/react-icons/dist/esm/createIcon';
+import {
+    ImageVulnerabilityCounter,
+    ImageVulnerabilityCounterKey,
+} from '../hooks/useImageVulnerabilities';
 
 export type BySeveritySummaryCardProps = {
     title: string;
-    severityCounts: Record<VulnerabilitySeverity, number>;
+    severityCounts: ImageVulnerabilityCounter;
     hiddenSeverities: Set<VulnerabilitySeverity>;
 };
 
-const severitiesCriticalToLow = [
-    'CRITICAL_VULNERABILITY_SEVERITY',
-    'IMPORTANT_VULNERABILITY_SEVERITY',
-    'MODERATE_VULNERABILITY_SEVERITY',
-    'LOW_VULNERABILITY_SEVERITY',
-] as const;
+const vulnCounterToSeverity: Record<ImageVulnerabilityCounterKey, VulnerabilitySeverity> = {
+    low: 'LOW_VULNERABILITY_SEVERITY',
+    moderate: 'MODERATE_VULNERABILITY_SEVERITY',
+    important: 'IMPORTANT_VULNERABILITY_SEVERITY',
+    critical: 'CRITICAL_VULNERABILITY_SEVERITY',
+} as const;
+
+const severitiesCriticalToLow = ['critical', 'important', 'moderate', 'low'] as const;
```

Comment:
Reader observations, not necessarily change request:

1. Make names more parallel `severitiesCriticalToLow` here and the following in hook?

    ```ts
    export const imageVulnerabilityCounterKeys = ['low', 'moderate', 'important', 'critical'] as const;
   ```

2. Is the LowToCritical order ever used? At least some occurrences work either way, correct?
    * `imageVulnerabilityCounterKeys.forEach(…)`
    * `severitiesCriticalToLow.forEach(…)`

Reply:
The idea here was to create a local variable that enforced the order specific to this component. I had thought about just making the default order in the hook CriticalToLow, but didn't want to have reliance on an order that is pretty much arbitrary in all other use cases.